### PR TITLE
Fixes bug when table do not exists (join)

### PR DESCRIPTION
**Pull request by imton**
When table do not exists we should check before.

### CODE REVIEW

**Comment by edpaget** [comment]
Looks pretty good to me, just a couple small things. Can you add a test case for this code, and also squash your commits into a single one?

File: lib/postgres_ext/arel/4.1/visitors/postgresql.rb [hunk 1]
```diff
@@ -5,15 +5,17 @@ module Visitors
     class PostgreSQL
       private
 
-      def visit_Array o, a
+      def visit_Array o, a
```
Comment (danmcclain) [inline lib/postgres_ext/arel/4.1/visitors/postgresql.rb]: Please kill the whitespace at the end of this line

**Comment by danmcclain** [comment]
Agree with @edpaget on this PR

File: Gemfile [hunk 2]
```diff
@@ -3,10 +3,6 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in postgres_ext.gemspec
 gemspec
 unless ENV['CI']
-  if RUBY_PLATFORM =~ /java/
```
Comment (edpaget) [inline Gemfile]: @danmcclain I'm going to clean this PR up and merge it since it seems useful. Do you want this part in the final result?

**Comment by edpaget** [comment]
Merged here: https://github.com/DockYard/postgres_ext/commit/f88cbf323a32b71521824eb2f7743ae9d1258353

**Pull request closed without merging**